README.md

```diff
@@ -63,54 +63,60 @@ General rules follow; but we also have language-specific guides for [Ruby](/lang
 
 ## Commits
 
- - Commit as you go. Prefer small commits that describe a single change to monolithic commits.
- > Why? This helps to document your changes and also to break big projects down into small steps.
+- Commits should ordinarily be as small as possible while maintaining the code in a workable state. Generally, we try not to commit code when the project is in a broken state.
+> Why? Smaller commits help to document your changes and also to break big projects down into small steps.
 
- - Write commit messages that assume the committer as the subject, have a past-tense verb, and describe your change in customers' terms.
- > Why? Houston generates release notes from commit messages.
+- When committing, we strive to minimize "commit noise", i.e., changes that don't affect the code or state of the project, especially whitespace changes.
 
-    ```bash
-    # bad
-    git commit -m "update"
-   
-    # bad
-    git commit -m "Don't leak the nuclear launch codes"
-   
-    # good
-    git commit -m "Prevented unauthorized users from reading our launch codes"
-    ```
+- The commit message should contain 3 parts: a tag, a summary of the commit, and the time spent coding the changes.
 
- - Begin commit messages with a tag in square brackets.
- > Why? Houston uses these when generating release notes.
- >
- > Commits tagged `[feature]`, `[improvement]`, and `[fix]` will all be turned into
- > release notes that we show to customers. Commits tagged `[refactor]` and `[skip]`
- > will only be seen by developers.
- 
-    ```bash
-    git commit -m "[feature] ..."
-    ```
+- The tag is enclosed in square brackets and is one of: `[skip]`, `[feature]`, `[improvement]` or `[fix]`/`[bugfix]`. These help Houston to know how to generate release notes once the feature is deployed.
+  - `[feature]` indicates that this commit introduces a new, user-facing feature. The commit summary will be the basis of describing this feature to the user and should be written with that in mind.
+  - `[improvement]` indicates that this commit significantly improves an existing user-facing feature. Its summary will likewise be incorporated into the release notes and should be written with that in mind.
+  - `[fix]`/`[bugfix]` indicates that this commit fixes a significant, user-facing bug. As above, the commit summary will be incorporated into the release notes upon deploy.
+  - `[skip]` indicates that the feature introduced or bug(s) fixed in the commit are either not user-facing or terribly significant to the user. The summary from a commit tagged with `[skip]` will _not_ be included in the release notes.
+  - `[squash]` is an experimental tag and indicates that this commit should ultimately be squashed into a previous commit before being merged into the base branch.
+  - `[wip]` is a special commit tag that is only to be used for a commit that is a work-in-progress. These commits should _always_ be popped off or amended and the tag replaced with one of the tags above.
```

Comment:
Maybe work this into what was there, more.

Don't delete this part:

```diff
- - Begin commit messages with a tag in square brackets.
- > Why? Houston uses these when generating release notes.
- >
- > Commits tagged `[feature]`, `[improvement]`, and `[fix]` will all be turned into
- > release notes that we show to customers. Commits tagged `[refactor]` and `[skip]`
- > will only be seen by developers.
- 
-    ```bash
-    git commit -m "[feature] ..."
-    ```
```

Our style guide should use the terse, imperative sentences (e.g. **Begin commit messages with a tag in square brackets.**) over third-person, passive ones (e.g. **The tag is enclosed in square brackets ...**)

This is clearer, direct, and takes less time to read :smile: